yarn-project/ethereum.js/src/retry/index.ts

```diff
@@ -1,6 +1,13 @@
 import { sleep } from '../sleep/index.js';
 import { Timer } from '../timer/index.js';
 
+/**
+ * Generates a backoff sequence iterator for exponential backoff with pre-defined values.
+ * The iterator will yield a series of time intervals (in seconds) to be used for backing off
+ * in error handling scenarios. The sequence is [1, 1, 1, 2, 4, 8, 16, 32, 64] and will not exceed 64.
+ *
+ * @yields {number} The next value in the backoff sequence.
+ */
 export function* backoffGenerator() {
```

Comment:
Some of these utilities are redundant, likely better off deduplicated with the foundation package, but that is best a separate PR